src/main/java/frc/robot/constants/IDs.java

```diff
@@ -10,4 +12,10 @@ public class IDs {
 		PowerDistribution.ModuleType.kRev
 	);
 
+	// ! need to check those IDs which doesn't exists yet since we don't have a working robot
+
+	public static final SparkMaxDeviceID ELEVATOR_FIRST_MOTOR = new SparkMaxDeviceID(0, CANSparkLowLevel.MotorType.kBrushless);
+
+	public static final SparkMaxDeviceID ELEVATOR_SECOND_MOTOR = new SparkMaxDeviceID(0, CANSparkLowLevel.MotorType.kBrushless);
```

Comment:
maybe upper/lower if its like this? i dont really know how the elevator works

Reply:
they are on each side of the elevator, this is fine by me, left and right if you want to change

Reply:
k